plugin.video.nos/resources/lib/requirements.txt

```diff
@@ -1,4 +1,3 @@
-xbmcswift2
 requests
```

Comment:
Please don't submit this file at all to the Kodi repository

Reply:
Cheers, removed!